user-data.sh

```diff
@@ -14,6 +14,9 @@ EOL
 cp -R /home/${user}/.ssh/* /root/.ssh/
 chown -R ${user}:${user} /home/${user}/.ssh/
 
+while sudo fuser /var/{lib/{dpkg,apt/lists},cache/apt/archives}/lock >/dev/null 2>&1; do
```

Comment:
No need for sudo.
Why do you need it?

Reply:
I will remove sudo, but I need it, its failed on apt lock